Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [package]
 name = "aardvark-dns"
+# This version specification is reused by .packit.sh to generate rpm version
```

Comment:
Note: Renovate will muck with these values.  Assuming packit runs in all PRs, that will include the renovate proposed-updates as well.  If this is a concern, it's possible to teach renovate to ignore certain dependencies and never try to update them.

Reply:
do you have an example in mind?

Reply:
Renovate manages dependency versions.  It periodically checks dependencies using language-idiomatic methods, i.e. cargo.conf for rust.  When it finds an update, it modifies the file to bump the version number, and opens a PR with the change.

I mentioned it here because Renovate will most certainly not read or respect your comment :wink: So if modifying a version in this file could have unintended packit-consequences, I need to deal with that in the renovate configuration.  Is this okay, or do we need to inform renovate to keep it's hands off some dependency or another?

Reply:
ah I see, let me send another PR to fix up the comments, there are some in .packit.sh too.